javaparser-core/src/main/java/com/github/javaparser/ast/stmt/TryStmt.java

```diff
@@ -57,7 +58,7 @@
  */
 public final class TryStmt extends Statement {
```

Comment:
Should this be named TryWithResourcesStatement ? Or are we using one statement for the TryStatement and the TryWithResources statement?

Reply:
There's only one indeed.

Reply:
I would think we should have two separate statements but I realize this would be a breaking change

Reply:
Hm, this is arbitrary: the try body, the final block and the catch statements are all optional, but we don't make separate nodes for those cases?